src/functionalTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerControllerTest.java

```diff
@@ -140,13 +159,13 @@ public void should_succeed_and_create_a_task_with_single_categories() {
             .body("hasWarnings.value", is(false))
             .body("warningList.value", is("[]"));
 
-        taskToTearDown = taskId;
     }
 
     @Test
     @Ignore("non-existing requirement for IA")
```

Comment:
If tests are being ignored, surely it would be better to delete the test completely? It's always in git history if we need it again ? I don't see the point of modifying code that isnt run

Reply:
We have another ticket for ignored tests